devices/HapticGlove/test/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+cmake_minimum_required(VERSION 3.5)
+project(SenseGloveTestApplication)
+
+find_package(IWear REQUIRED)
+find_package(YARP REQUIRED)
+find_package(iDynTree REQUIRED)
+
+
+
+add_executable(${PROJECT_NAME} VisualizeFrames.cpp)
+
+target_link_libraries(${PROJECT_NAME} PUBLIC
+    IWear::IWear
+    YARP::YARP_os
+    YARP::YARP_sig
+    YARP::YARP_dev
+    YARP::YARP_math
+    iDynTree::idyntree-visualization
+    )
+
+install(TARGETS ${PROJECT_NAME} DESTINATION bin)
+
+install(FILES config.ini DESTINATION bin )
```

Comment:
Are we sure we want to have a `bin/config.ini`? I suspect this might create a lot of problems because it would appear in the command autocompletion of any terminal.

Reply:
I added here, since I had in mind this code as a test code and not installing it inside superbuild. But since, now, it can be used to visualize the frames of any wearable data/device, I was thinking to move this snippet as a module to `wearable/modules/IWearFrameVisualizer`. What do you think?
cc @Yeshasvitvs @lrapetti

Reply:
Yep! I'm not sure where these configs are typically installed by wearables, I expect somewhere in `CMAKE_INSTALL_DATADIR` (share/).

Reply:
I have updated the it as we decided. I added the `modules` folder, then I installed it on the directory you suggested correctly. Let me know if this is OK.

Reply:
Looks good!